fe/fe-core/src/main/java/org/apache/doris/planner/TableFunctionNode.java

```diff
@@ -133,12 +146,6 @@ public String getNodeExplainString(String prefix, TExplainLevel detailLevel) {
             return output.toString();
         }
 
-        output.append(prefix + "tuple id: ");
```

Comment:
Why remove this?

Reply:
The code in the parent class is duplicated.